src/containers/NamespacesDropdown/NamespacesDropdown.js

```diff
@@ -21,16 +21,33 @@ const NamespacesDropdown = props => {
   return <TooltipDropdown {...props} />;
 };
 
+const allNamespacesLabel = 'All Namespaces';
+
 NamespacesDropdown.defaultProps = {
+  allNamespacesLabel,
   items: [],
   loading: true,
-  label: 'Select Namespace'
+  label: 'Select Namespace',
+  showAllNamespaces: false
```

Comment:
Why is this field needed ?

Reply:
We only want to show 'All Namespaces' in the nav dropdown. Other places the namespace dropdown is used should not display this option.